### PR TITLE
New version: GitHub.GitLFS version 3.3.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Git LFS | Git LFS version 3.3.0    |
| Version     | 3.3.0     | 3.3.0 |
| Publisher   | GitHub, Inc.   | GitHub, Inc.      |
| ProductCode |           | {286391DE-F778-44EA-9375-1B21AAA04FF0}_is1    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90133)

### CODE REVIEW

**Review by jedieaston — DISMISSED** [review]

**Comment by stephengillie** [comment]
Git is a dependency, apparently. Is there a version of Git already in Winget that would work in this scenario? 
image

**Comment by stephengillie** [comment]
Found a possible cause: It appears that the GitLFS installer finds Git via Windows Path. A PowerShell window's $env:path variable, and the PATH variable in a Command Prompt window, aren't updated while it's running - normally only at launch. I was able to complete installing this package in a VM by installing it with Winget, accepting the error, closing and reopening either PowerShell or a Command Prompt, then installing the package with Winget again. This should be an issue regarding dependencies and/or maybe how Winget handles the Windows Path - I'll look into reporting it.

File: manifests/g/GitHub/GitLFS/3.3.0/GitHub.GitLFS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: GitHub.GitLFS
+PackageVersion: 3.3.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+ReleaseDate: 2022-11-30
```
Comment (stephengillie) [inline manifests/g/GitHub/GitLFS/3.3.0/GitHub.GitLFS.installer.yaml:11]:
```suggestion
ReleaseDate: 2022-11-30
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Git.Git
```

**Review by stephengillie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**